README.md

```diff
@@ -12,25 +12,34 @@ Video: https://youtu.be/KiDnPsnLmLU
 
 # Notes/Howto
 
-To run demo with four cones use:
+There has been serious development to extend original code suited mainly for
+modified John Deere X300R garden tractor to more general tool. Now it runs
+on more than four machines, including autonomous boat. It provides logging
+and replay facility, so simple logging of multiple inputs is unified and
+stored in binary format with timestamps.
 
-```
-python ./navpat.py run config/playground15x5.json
-```
+The current code is under `osgar` directory and the remaining code is ported
+now. If you are using OSGAR directly from sources please make sure that
+you setup PYTHONPATH to the root of this project.
+
+The example configurations are stored in `config` folder. JSON files are
+used.
+
+## Examples
+
+### Collect data from sensor(s)
+
+There is a `osgar/robot.py` to run data collection based on given configuration file.
+If you would like to collect GPS data available on serial port use modified version
+of `test-windows-gps.json` for Windows or `test-gps-imu.json` for Linux.
 
-To replay existing log file use meta log, for example:
-```
-python ./navpat.py replay logs/meta_160821_160615.log
-```
 
-Current visualization is using **Eduro Viewer**.
-Add extra parameter **--view** in log replay to see the data
 ```
-python ./navpat.py replay logs/meta_160821_160615.log --view
+python ./osgar/robot.py run config/test-windows-gps.json
 ```
 
-For more options see
+To replay existing log use:
 ```
-python ./navpat.py --help
+python ./osgar/replay.py --module <module name> <log file name>
 ```
 
```

Comment:
Can we add a note howto control your own robot using osgar libraries? Where to look for examples? What is the minimum that needs to be done to make the new robot move?

Reply:
Well, yes, good point, ... but the "apps" are so far very specific to robot. The most generic example is with Eduro, which is not merged into master yet. Or would you recommend something? I would probably postpone it after merge and release 0.0.1

Reply:
I didn't mean apps. Having an app will not make it work with a new robot. I meant some howto like make a class inherited from Thread, in its run() method put while true cycle, listen to the bus, use existing transport like Serial or Socket etc. With the end result where the new robot can be controlled by any app in osgar.

Reply:
OK, I see. As discussed over e-mail, I would add it to ver2 and not to block 0.0.1 release. I am also not sure if it should be in the main README.md ... but is should be relatively short paragraph, so maybe yes ;-). For this PR, I will rebase and merge it, and return to it in next cycle/iteration. Thanks.